docs/user/security/rbac_tutorial.asciidoc

```diff
@@ -0,0 +1,114 @@
+[[space-rbac-tutorial]]
+=== Tutorial:  Use role-based access control to customize Kibana spaces
+
+With role-based access control (RBAC), you can provide users access to data, tools,
+and Kibana spaces.  In this tutorial, you will learn how to configure roles
+that provide the right users with the right access to the data, tools, and
+Kibana spaces.
+
+[float]
+==== Scenario
+
+Our user is a web developer working on a bank's
+online mortgage service.  The web developer has these 
+three requirements:
+
+* Have access to the data for that service 
+* Build visualizations and dashboards
+* Monitor the performance of the system
+
+You'll provide the web developer with the access and privileges to get the job done.
+
+[float]
+==== Prerequisites
+
+To complete this tutorial, you'll need the following:
+
+*  **Administrative privileges**: You must have a role that grants privileges to create a space, role, and user. See the {ref}/built-in-roles.html[built-in] roles.
```

Comment:
**Question** What do you think about expanding this just a little, so users don't have to fill in the blanks themselves?

> You must have a role that grants privileges to create a space

Such as the `kibana_user` role

> You must have a role that grants privileges to create a ... role, and user

This is any role which grants the `manage_security` cluster privilege. The only built-in role which satisfies this is the `superuser` role, but users don't need that level of access to complete this tutorial.


The superuser would satisfy both requirements though, so if we're looking for an "easy" getting started experience, then this might be the way to go. However, if we're trying to educate on best practices, then we shouldn't advise using the superuser all over the place.

Reply:
@gchaps , can you discuss why we avoid superuser / elastic user here?

Reply:
It's because of what @legrego said--we don't want to advise using superuser all over the place.

However, it sounds like its needed here.  I think the text should include the part about the `manage_security` cluster privilege and that the built-in `superuser` role meets this need.

Reply:
Groovy, it will reduce the head-scratching.  Adding now.